scripts/simulate.py

```diff
@@ -0,0 +1,192 @@
+"""
+CLI for hera_sim
+"""
+import argparse
+import copy
+import os
+import sys
+import warnings
+import yaml
+
+import hera_sim
+from hera_sim import cli_utils
+from pyuvsim.simsetup import _parse_layout_csv
+from astropy.coordinates import Angle
+from astropy import units
+try:
+    import bda
+    from bda import bda_tools
+except ImportError:
+    warnings.warn("bda failed to import. Baseline-dependent averaging not available.")
```

Comment:
This might get annoying -- may just want to use an appropriate error if/when bda is actually _used_.

Reply:
I think I can maybe just remove the `warnings.warn(...)` statement, since there is a step in the CLI that checks to make sure `bda` is installed if `bda_params` is specified in the configuration and errors if it isn't.